scheduling-api-graphql/scheduling-api-graphql-common/src/main/java/org/ow2/proactive/scheduling/api/graphql/common/Arguments.java

```diff
@@ -39,7 +39,9 @@
     BEFORE,
     FILTER,
     FIRST,
-    LAST;
+    LAST,
```

Comment:
these new filters should not added here. They are going to be used inside the Filter body